src/seedu/addressbook/parser/Parser.java

```diff
@@ -73,33 +74,36 @@ public Command parseCommand(String userInput) {
 
         switch (commandWord) {
```

Comment:
The keywords `switch` and `case` should be at the same level of indentation. Coding Standard Point 10